db/connection.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+require('dotenv/config');
+
+mongoose.connect(process.env.ATLAS_URL,{useNewUrlParser: true,useUnifiedTopology: true},(e)=>{
+    if(e){
+        throw console.error(`Database connection error -> ${e}`);
```

Comment:
Using throw will force the app to crash in case there is no connection set for any reason, it is better we console the error `only`.